### DISCUSSION TITLE
Sending logs to 2 Managements

### DISCUSSION THREAD

**Discussion by itdeveloper50** [Security-Onion-Solutions/securityonion#9174] [Unsupported Versions]
I have Wazuh-agent installed on my hosts.
Is it possible to send security logs to 2 (two) Managements (to 2 IPs)?

**Comment by dougburks** [comment]
One option might be to have your Wazuh agent send to a single Wazuh manager but then have that Wazuh manager forward the logs to a separate system:
https://docs.securityonion.net/en/2.3/logstash.html#forwarding-events-to-an-external-destination
